tests/keras/backend/backend_test.py

```diff
@@ -128,7 +132,7 @@ def check_single_tensor_operation(function_name, x_shape_or_val, backend_list, *
         assert_list_with_ref(z_list, assert_value_with_ref)
     else:
         assert_list_pairwise(z_list, allclose=assert_value_equality)
-    assert_list_keras_shape(z_list)
```

Comment:
Originally ```z``` is the output of ```K.eval``` which is a numpy ndarray usually. It doesn't have ```_keras_shape``` attribute, so not trigger the inside check.
The fix is: we pass into two list, one is tensor list, the other is output value by ```K.eval```. We check the equality between ```tensor._keras_shape``` and ```value.shape```.